### PR TITLE
fix: hide disabled dislike

### PR DESCRIPTION
**Pull request by FineFindus**
Hides the dislikes icon and text when if they are disabled (API returns -1). 

| Before                                                                                                                                     	| After                                                                                                                                      	|
|--------------------------------------------------------------------------------------------------------------------------------------------	|--------------------------------------------------------------------------------------------------------------------------------------------	|
| Screenshot of a video showing -1 dislikes 	| Screenshot of a video showing no dislikes 	|

### CODE REVIEW

File: app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt [hunk 1]
```diff
@@ -885,7 +885,11 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
             playerViewsInfo.text = getString(R.string.normal_views, views, localizeDate(streams))
 
             textLike.text = streams.likes.formatShort()
-            textDislike.text = streams.dislikes.formatShort()
+            if (streams.dislikes >= 0) {
+                textDislike.visibility = View.VISIBLE
```
Comment (Bnyro) [inline app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt:889]:
```suggestion
                textDislike.isVisible = true
```

**Comment by Bnyro** [comment]
This does not yet cover when you play multiple videos:
* The first one has a dislike count available.
* The second one doesn't

-> The second video will show a dislike count, although it's -1

**Comment by FineFindus** [comment]
What do you mean by playing multiple videos? Please include steps on how to reproduce it.

**Comment by Bnyro** [comment]
It's not reproducible atm because Return YouTube dislike doesn't work.

File: app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt [hunk 2]
```diff
@@ -885,7 +885,11 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
             playerViewsInfo.text = getString(R.string.normal_views, views, localizeDate(streams))
 
             textLike.text = streams.likes.formatShort()
-            textDislike.text = streams.dislikes.formatShort()
+            if (streams.dislikes >= 0) {
+                textDislike.isVisible = true
+                textDislike.text = streams.dislikes.formatShort()
+            }
```
Comment (Bnyro) [inline app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt:891]:
```suggestion
            textDislike.isVisible = streams.dislikes >= 0
            textDislike.text = streams.dislikes.formatShort()
```

**Review by Bnyro — APPROVED** [review]

**Pull request merged**